ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -53,28 +54,27 @@ public struct FluentButton: View {
         state.text = text
         state.image = image
         self.state = state
-        self.tokens = state.tokens
+
+        // We want separate lookup keys for each permutation of `style` and `size`.
+        self.tokenKey = "\(type(of: self))_\(style.rawValue)_\(size.rawValue)"
```

Comment:
Is there going to be a pattern for the controls going forward to identify the token keys?
I feel like we should have a standard for these.

Reply:
Agreed, this is getting a little out of hand. I'm thinking instead of returning a string I'll have this API return an array of `Any` values, and then concat them together in the caller.